velox/docs/functions/presto/array.rst

```diff
@@ -214,6 +214,14 @@ Array Functions
     Returns a subarray starting from index ``start``(or starting from the end
     if ``start`` is negative) with a length of ``length``.
 
+.. function:: sequence(start, end) -> array
```

Comment:
end -> stop.
Should we just copy the description from Presto documentation, seems like it is more clear there?
https://prestodb.io/docs/current/functions/array.html#id5